cmd/kubeadm/app/util/output/output.go

```diff
@@ -133,7 +141,11 @@ func NewOutputFlags(textPrintFlags TextPrintFlags) *PrintFlags {
 type Printer interface {
 	PrintObj(obj runtime.Object, writer io.Writer) error
 	Fprintf(writer io.Writer, format string, args ...interface{}) (n int, err error)
+	Fprintln(writer io.Writer, args ...interface{}) (n int, err error)
 	Printf(format string, args ...interface{}) (n int, err error)
```

Comment:
should we have a Println(), given we have some printf("....\n") cases?
e.g. cmd/kubeadm/app/phases/upgrade/compute.go